docs/hub/gguf-llamacpp.md

```diff
@@ -1,6 +1,30 @@
 # GGUF usage with llama.cpp
 
-Llama.cpp allows you to download and run inference on a GGUF simply by providing a path to the Hugging Face repo path and the file name. llama.cpp download the model checkpoint and automatically caches it. The location of the cache is defined by `LLAMA_CACHE` environment variable, read more about it [here](https://github.com/ggerganov/llama.cpp/pull/7826):
+NEW: You can now deploy any llama.cpp compatible GGUF on Hugging Face Endpoints, read more about it [here](https://huggingface.co/docs/inference-endpoints/en/others/llamacpp_container)
```

Comment:
I'm wondering if we support `[!TIP]` blockquote like on github?

https://github.com/orgs/community/discussions/16925

Reply:
afaik yes